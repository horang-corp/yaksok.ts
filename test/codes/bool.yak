만약 (1 < 3) 이거나 (3 < 1) 이면
    "OR 연산자는 둘 중 하나만 참이어도 참이다." 보여주기

만약 (1 < 3) 이고 (3 > 1) 이면
    "AND 연산자는 둘 다 참이어야 참이다." 보여주기