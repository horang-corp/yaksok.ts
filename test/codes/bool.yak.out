OR 연산자는 둘 중 하나만 참이어도 참이다.
AND 연산자는 둘 다 참이어야 참이다.
